internal/fs/local_file_test.go

```diff
@@ -857,3 +857,44 @@ func (t *LocalFileTest) TestStatLocalFileAfterRecreatingItWithSameName() {
 	ExpectEq("test.txt", f.Name())
 	ExpectFalse(f.IsDir())
 }
+
+func (t *LocalFileTest) TestStatSucceedsOnNewFile() {
+	t.serverCfg.NewConfig = &cfg.Config{
```

Comment:
Is this test method same as statOnLocalFile() ?

Reply:
Yes, removed it now. Thanks!